### DISCUSSION TITLE
Calling peaks with group.by or creating a common peak set?

### DISCUSSION THREAD

**Discussion by klgoss** [stuart-lab/signac#1490] [Q&A] [answered]
I have several multiome datasets (same experiment, different timepoints so 12 total datasets). I originally merged them, created a common peak set (following this vignette: https://stuartlab.org/signac/articles/merging#creating-a-common-peak-set). I also saw where you could merge objects first, then do CallPeaks() and use the "group.by" parameter. Since my data isn't necessarily different datasets, it was all one big experiment, would it be a better approach to call peaks using the latter approach? I'm pretty new to ATACseq so any insight is appreciated.

**Comment by timoast** [comment, marked as answer]
We demonstrated some advantages of peak calling in the [Signac paper](https://www.nature.com/articles/s41592-021-01282-5), mainly that MACS2 can do better than the cellranger peak calls (although cellranger-atac has now updated their peak calling algorithm so this may not be an issue), and that peak calling on a per-celltype basis can help to identify peaks that are unique to low-abundance cell types. If you think either of these things could help in your analysis, certainly try per-celltype peak calling with MACS2 using the CallPeaks function.

**Comment by NadiaEdel** [comment]
Hello! I am also working with 10x multiome datasets. I am struggling with creating the common peak set. I am following the vignette: https://stuartlab.org/signac/articles/merging#creating-a-common-peak-set, but I get stuck in the "Create fragment object" part where it asks for the "pbmc500/atac_pbmc_500_nextgem_singlecell.csv" file. I do not have that type of file in my own outs folder, the closest I see is the "per_barcode_metrics". Am I missing something? Is there a way to create that singlecell.csv file? I apologise if the question has been made before or if it is rather silly...but I am stuck in the common set part (I am working on a Windows OS, so I cannot run MACS2 -I tried with Ubuntu from WSL but no luck so far-). In the outs folder I have both peaks.bed and fragments.tsv kind-of-files.
Thank you!
